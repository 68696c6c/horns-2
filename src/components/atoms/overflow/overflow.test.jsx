/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Overflow from '.'

describe('Overflow', () => {
  it('should render as default', () => {
    const { container } = render(
      <Overflow height="400px" x="scroll" y="scroll" />
    )
    expect(container).toMatchSnapshot()
  })
})
