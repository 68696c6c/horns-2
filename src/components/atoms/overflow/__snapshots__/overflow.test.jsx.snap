// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overflow should render as default 1`] = `
.emotion-0 {
  width: 100%;
  height: 400px;
  overflow-x: scroll;
  overflow-y: scroll;
}

<div>
  <div
    class="emotion-0"
    height="400px"
    x="scroll"
    y="scroll"
  />
</div>
`;
